code/game/gamemodes/wizard/wizard.dm

```diff
@@ -378,9 +439,24 @@ Made a proc so this is not repeated 14 (or more) times.*/
 						if(I.icon_state == "wizard")
 							wizard_mind.current.client.images -= I
 
+		for(var/datum/mind/apprentice_mind in apprentices)
+			if(apprentice_mind.current)
+				if(apprentice_mind.current.client)
+					for(var/image/I in apprentice_mind.current.client.images)
+						if(I.icon_state == "wizard" || I.icon_state == "apprentice")
```

Comment:
Honestly I have no idea how this works, it's just rev code with the names changed